### ISSUE TITLE
Passing an empty file throws error

### ISSUE THREAD

**Issue by OmgImAlexis** [dzek69/brotli-cli#6]
`./dist/assets/home.8e212dc3.css` is empty and has `0 bytes`.

```
> brotli-cli compress --glob './dist/*.html' --glob './dist/*.ico'  --glob './dist/assets/*.css' --glob './dist/assets/*.js'

Error happened during compression process, the process is stopped.
File: ./dist/assets/home.8e212dc3.css
TypeError: Empty response returned from brotli
    at /Users/xo/code/OmgImAlexis/siteli.st/client/node_modules/brotli-cli/dist/index.js:146:27
    at async Promise.all (index 7)
    at async Object.handler (/Users/xo/code/OmgImAlexis/siteli.st/client/node_modules/brotli-cli/dist/index.js:133:9)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @0.0.0 compress-assets: `brotli-cli compress --glob './dist/*.html' --glob './dist/*.ico'  --glob './dist/assets/*.css' --glob './dist/assets/*.js'`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @0.0.0 compress-assets script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/xo/.npm/_logs/2021-07-11T02_23_18_003Z-debug.log
```

**Comment by OmgImAlexis** [comment]
https://github.com/dzek69/brotli-cli/blob/32c2693800e754050293bf049b22fe7740ad47d7/src/index.ts#L189-L191

Maybe a flag like `--skip-empty` and that'd change this from throwing an error to using `return`?

**Comment by dzek69** [comment]
Fixed on 1.0.3
